<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signupUi</class>
 <widget class="QDialog" name="signupUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lab_up_rpt_pwd">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>438</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 14pt &quot;Ubuntu Condensed&quot;;</string>
   </property>
   <property name="text">
    <string>           Repeat your password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="up_pwdlineEdit">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>401</y>
     <width>250</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(252, 253, 149);
</string>
   </property>
   <property name="text">
    <string>123qwe!@#QWE</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="tosignin">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>544</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(32, 74, 135);</string>
   </property>
   <property name="text">
    <string>Sign in</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lab_newto">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>544</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Has an account?</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>32</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Bebopw.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="signup">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>603</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>TakaoPGothic</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);
background-color: qradialgradient(spread:pad, cx:0.479, cy:0.5, radius:2, fx:0.479, fy:0.5, stop:0.195876 rgba(0, 150, 29, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string>Sign up for Bebop</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="up_email_lineEdit">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>313</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(252, 253, 149);
</string>
   </property>
   <property name="text">
    <string>249691892@qq.com</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lab_up_pwd">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>345</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 14pt &quot;Ubuntu Condensed&quot;;</string>
   </property>
   <property name="text">
    <string>           Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_up_email">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>267</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 14pt &quot;Ubuntu Condensed&quot;;</string>
   </property>
   <property name="text">
    <string>           Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="up_rpt_pwdlineEdit">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>486</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(252, 253, 149);
</string>
   </property>
   <property name="text">
    <string>123qwe!@#QWE</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="up_usrlineEdit">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>223</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(252, 253, 149);
alternate-background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>matrix98</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lab_up_usr">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>177</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 14pt &quot;Ubuntu Condensed&quot;;</string>
   </property>
   <property name="text">
    <string>           Username</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
